### ISSUE TITLE
Other ways of accessing applet-based equipment

### ISSUE THREAD

**Issue by seneca-c** [netinvent/ipmi-starter#3]
Another approach to applet-based equipment interfaces is what Dell provides in their "VNX launcher" package.  It's an older, 32-bit version of Firefox Portable (I they used Firefox 51) packaged up with 32-bit Java.  I needed to replace the included copy of Java 8 with the version of Java 6 that I used with Java Web Start, but it works.  I have yet to be annoyed enough by the read-only profile to take the measures necessary to get it to actually remember that I already clicked through all the security prompts.

**Comment by deajan** [comment]
Indeed, I used to keep old portable navigators with java installed for that purpose too ;)
Note that you can configure your java (see the readme.md file) to avoid your clicking hell.
